src/main/java/racing/controller/GameController.java

```diff
@@ -0,0 +1,40 @@
+package racing.controller;
+
+import racing.domain.RacingGame;
+import racing.util.Converter;
+import racing.validate.InputVerifier;
+import racing.view.input.InputView;
+import racing.view.output.OutputView;
+
+import java.util.List;
+
+public class GameController {
+    private final RacingGame racingGame;
+
+
+    public GameController() {
+        this.racingGame = new RacingGame(initializeCarNames(), initializeTryCount());
+    }
```

Comment:
GameController는 console RacingGame 만 지원하는걸까요?! 
이부분은 다른분 [피드백](https://github.com/woowacourse/java-racingcar/pull/473#discussion_r1102245243)에서도 남겼는데요!! 

현재 구조에선, RacingGame의 인스턴스를 main에서 가져와도 좋을 것 같아요 :) 

```suggestion
    public GameController(RacingGame racingGame) {
        this.racingGame = racingGame;
    }
```

Reply:
RacingGame의 인스턴스인 carNames와 tryCount를 main단에서 가져오게 변경해 봤습니다!